wait_for.sh

```diff
@@ -120,6 +121,16 @@ get_service_state() {
     echo "$get_service_state_states"
 }
 
+# Terraform cloud Workspace runStatus
+# ready when 'applied'
+get_workspace_state() {
+    get_workspace_state_name="$1"
+    get_workspace_state=$(kubectl get workspace "$get_workspace_state_name" -o jsonpath='{.status.runStatus}')
+    if [ "x${get_workspace_state}" != "xapplied" ]; then
+	echo "$get_workspace_state"
```

Comment:
we may want to limit the number of the logs (for example echo every 1 of 10 times) if it takes a while for the workspace to come up. Or add a wait after the echo.

Reply:
I believe that is already handled via `wait_for_resource` which calls this fn.